mod/01_rating/spec/set/right/type_search_spec.rb

```diff
@@ -1,22 +1,25 @@
 # -*- encoding : utf-8 -*-
 describe Card::Set::Right::TypeSearch do
-  describe "filter_and_sort view" do
-    context "topic" do
-      it "show filter_and_sort items" do
+  describe 'filter_and_sort view' do
+    context 'topic' do
+      it 'show filter_and_sort items' do
         topic = get_a_sample_topic
         company = get_a_sample_company
-        claim = create_claim "death is a joke", {"+company"=>{:content=>"[[#{company.name}]]"},"+topic"=>{:content=>"[[#{topic.name}]]"}}
+        create_claim 'death is a joke',
+                     '+company' => { content: "[[#{company.name}]]" },
+                     '+topic'   => { content: "[[#{topic.name}]]"   }
 
         type_search_card = Card.fetch "#{topic.name}+company+type_search"
         html = type_search_card.format.render_filter_and_sort
 
-        expect(html).to have_tag("div",:with=>{:class=>"yinyang-list"}) do
-          with_tag("div",:with=>{:class=>"yinyang-row","data-sort-name"=>"DEATH STAR"}) do
-            with_tag "div",:with=>{:id=>"Force+Death_Star+yinyang_drag_item"}
+        expect(html).to have_tag('div',:with=>{:class=>'yinyang-list'}) do
+          with_tag('div',:with=>{:class=>'yinyang-row','data-sort-name'=>'DEATH STAR'}) do
```

Comment:
Line is too long. [90/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space missing after comma.<br>Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.